formats/world_location_news_article/publisher/links.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "topical_events": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "available_translations": {
```

Comment:
`available_translations` gets added automatically by the publishing API:
https://github.com/alphagov/govuk-content-schemas/blob/master/lib/govuk_content_schemas/frontend_schema_generator.rb#L26